src/styles/components/service-table/styles.less

```diff
@@ -60,6 +60,15 @@
   .status-waiting-indicator {
     margin-left: @base-spacing-unit * 1/4;
   }
+
+  .quota-info {
+    margin-bottom: @base-spacing-unit;
+
+    span {
+      font-size: 14px;
```

Comment:
`InfoBoxInline` already sets a font-size which we override. sounds like we're introducing a special case or inconsistency here. do we have good reason for that?

Reply:
I tried to follow the design as close as possible.

Reply:
@mperrotti ☝️  ?

Reply:
@GeorgiSTodorov - don't change the font size.

In general, if a mockup uses a component it's styled slightly differently, always use the component style.